### DISCUSSION TITLE
利用AI創作的藝術家

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#688] [General]
最近看到很多人在討論AI會不會取代藝術家的問題，
前陣子我有看到一個TED的影片，連結在這：https://www.youtube.com/watch?v=UxQDG6WQT5s
影片中這位藝術家Refik Anadol是一名土耳其裔美國人的新媒體藝術家和設計師，他的作品大多由數據驅動的機器學習演算法組成，利用這些演算法創建抽象的，夢幻般的環境。
看完這部影片我覺得雖然AI無法擁有人類藝術家的創造力，但藝術家能透過AI做出人類無法完成的作品，對於面對AI時代來臨的藝術家，是很好的應變！

在此附上Refik Anadol的作品集：https://refikanadol.com

**Comment by 0212Jerry** [comment]
我認為或許藝術在有Ai 存在的情況下可能會多出一個以Ai 為主的類別，畢竟Ai 在藝術方面實在太多爭議，或許以兩個不同的立場來看待會比爭論還要好。

**Comment by Vincent20030314** [comment]
AI作圖很方便，但這樣可能導致未來人類的創作依靠AI，進而降低人類創造力。

**Comment by HsuChen88** [comment]
我覺得利用AI的技術來製圖可以提升作圖的效率，但最原始的圖畫構想得是人的產物才有意義。

**Comment by yjpxi** [comment]
我是覺得AI 可以成為藝術家的助手，提供靈感、建議或協助完成技術性繁瑣的部分，這樣藝術家不僅可以專注於創意的核心，而非技術操作，進而創作出更加豐富和深刻的作品。

**Comment by unnamed0604** [comment]
藝術家若能配合AI一同創作，或許能讓某些藝術家跳脫原先舒適圈，比如使擅長畫風景畫的畫家能結合AI給予的幫助產出抽象畫作也說不定
